src/pages/workspaces/workspace/WorkspaceContainer.js

```diff
@@ -350,6 +350,15 @@ export const wrapWorkspace = ({ breadcrumbs, activeTab, title, topBarContent, sh
       }
     })
 
+    // TODO [WOR-750] We expect a transient error while the workspace is cloning. This code
+    // ignores any error and does not retry; future changes will replace this logic with retries.
+    const loadAzureLocation = withErrorIgnoring(async workspace => {
```

Comment:
Note to reviewers: this temporary code will be removed when https://github.com/DataBiosphere/terra-ui/pull/3765 merges

Reply:
The PR has now merged so this branch needs to be rebased to pick up the changes.

Reply:
Great, I have removed Workspaces as a reviewer from this PR because I do not think you have edited any Workspaces-owned files at this point. You should get a second reviewer from the IA team.